### DISCUSSION TITLE
Button to zoom to location when adding "Thing"

### DISCUSSION THREAD

**Discussion by Aggaar** [streetcomplete/StreetComplete#5734] [Ideas]
I offen find myself zooming in an potitioning the "Thing" marker over my potition marker when mapping.
For example when adding a bench i would hold my phone over the center of the bench, and then just add the marker wherever my potition is on the map.
I think it would be helpful if there were an option to just add your "Thing" to whatever the coordinats of your device is.
I don't know if a new button to center the map, or a setting to automatically zoom to the device coordinats when adding a "Thing" is the best implementation, but I think it would be helpfull.
(Moving the "Thing" marker to the middle of the screen would altso work, then you would jut have to click center map, and then add your thing)

**Comment by HolgerJeromin** [comment]
GPS (device) position can be far of. Compare with the blue line which is quite wobbly.
So better to manually place the thing compared to streets and buildings.
